### PR TITLE
Migrate from travis/cix to cirrus-ci

### PR DESCRIPTION
**Pull request by tomverin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-install: true
-jdk: openjdk11
-git:
-  depth: false
-script: "./travis.sh"
-cache:
-  directories:
-  - "$HOME/.sonar"
-  - "$HOME/.gradle/caches/"
-  - "$HOME/.gradle/wrapper/"
```
Comment (pynicolas) [inline .travis.yml]: Should we stop caching that folder?

File: .cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+setup_gradle_cache_template: &SETUP_GRADLE_CACHE
+  gradle_cache:
+     folder: ${GRADLE_USER_HOME}/caches
+  create_gradle_directory_script:
+    - mkdir -p "${GRADLE_USER_HOME}"
+
+cleanup_gradle_cache_script_template: &CLEANUP_GRADLE_CACHE_SCRIPT
+  cleanup_gradle_script:
+    - /usr/bin/find "${GRADLE_USER_HOME}/caches/" -name "*.lock" -type f -delete
+    - rm -rf "${GRADLE_USER_HOME}/caches/5.4.1/"
```
Comment (pynicolas) [inline .cirrus.yml]: Why `5.4.1`? Is it going to change in the future? Should we use a regular expression here?

Comment (tomverin) [reply .cirrus.yml]: that's the version specfically used here https://github.com/SonarSource/sonar-jacoco/blob/master/gradle/wrapper/gradle-wrapper.properties#L5
so when updating the wrapper this script should be updated too
don't know about a regexp that could help us here

Comment (pynicolas) [reply .cirrus.yml]: What about something like the following?
```ls "${GRADLE_USER_HOME}/caches | grep -E "([0-9]+\\.)+[0-9]+" | xargs rm -rf```

Comment (tomverin) [reply .cirrus.yml]: this does not work

**Pull request merged**